### PR TITLE
Add workflow for new open GitHub pull requests

### PR DESCRIPTION
**Pull request by Amdrel**

### CODE REVIEW

**Review by dustinsgoodman — DISMISSED** [review]

File: .github/workflows/pull-request-opened-notify-slack.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Send a Slack notification when PRs are opened
```
Comment (dustinsgoodman) [inline .github/workflows/pull-request-opened-notify-slack.yml:1]: could add this is to alert someone for review

File: .github/workflows/pull-request-opened-notify-slack.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+on:
+  pull_request:
+    types:
+      - opened
+jobs:
+  send-slack-notification:
+    if: github.event.pull_request.merged == false &&
+      github.event.pull_request.user.login != 'dependabot[bot]'
```
Comment (dustinsgoodman) [inline .github/workflows/pull-request-opened-notify-slack.yml:9]: might be worth actually listening for dependabot but that's not even in this repo

File: .github/workflows/pull-request-opened-notify-slack.yml (hunk 1 above)
Comment (Amdrel) [reply .github/workflows/pull-request-opened-notify-slack.yml:1]: I'm not sure what you mean here. Do you mean adding a ping to the Slack message itself?

File: .github/workflows/pull-request-opened-notify-slack.yml (hunk 2 above)
Comment (Amdrel) [reply .github/workflows/pull-request-opened-notify-slack.yml]: Adjusted :+1:

File: .github/workflows/pull-request-opened-notify-slack.yml (hunk 1 above)
Comment (dustinsgoodman) [reply .github/workflows/pull-request-opened-notify-slack.yml:1]: Changing this to say "Send a Slack notification when PRs are opened to alert of that a review is needed". We care and we want people know that's why this is here so they're not suspicious about why there is a slack ping.

Comment (Amdrel) [reply .github/workflows/pull-request-opened-notify-slack.yml]: Oh gotcha, makes sense! I pushed a commit that changes that text

**Review by dustinsgoodman — APPROVED** [review]

**Pull request merged**